src/Illuminate/Contracts/Queue/Queue.php

```diff
@@ -96,4 +96,19 @@ public function getConnectionName();
      * @return $this
      */
     public function setConnectionName($name);
+
+    /**
+     * Set the queue prefix.
+     *
+     * @param  string  $prefix
+     * @return $this
+     */
+    public function setQueuePrefix($prefix = null);
+
+    /**
+     * Get the queue prefix.
+     *
+     * @return string
+     */
+    public function getQueuePrefix();
```

Comment:
@ntzm since the method is in the abstract parent class, all the driver implementations will inherit it, so everything will work fine.  do people define implementations that don't extend the abstract class?

on principle I agree with you. some projects seem more lax when adding new methods to the interface. can an owner comment on this?

Is the simple solution to remove it from the contract for 5.4? as I stated no one is **required** to use the prefix, so we'd be fine without it.